core/src/main/java/com/datastax/oss/driver/api/core/AllNodesFailedException.java

```diff
@@ -75,6 +76,7 @@ private static String buildMessage(Map<Node, Throwable> errors) {
     return errors;
   }
 
+  @Nonnull
```

Comment:
Thee is no automatic tool to generate or suggest where to place annotations to my knowledge. IntelliJ can however suggest to propagate annotations from interfaces to implementations, but otherwise I've been crawling the entire codebase.